misk-launchdarkly/src/main/kotlin/misk/feature/launchdarkly/LaunchDarklyFeatureFlagsModule.kt

```diff
@@ -89,5 +94,6 @@ data class LaunchDarklyConfig @JvmOverloads constructor(
   @Redact
   val sdk_key: String,
   val base_uri: String,
+  val use_relay_proxy: Boolean? = true,
```

Comment:
What would null signify? Can it just be non-null?

Reply:
I think if I make it non-null, that would force somebody to provide that in the config right? Or have I misunderstood?

I was trying to avoid anybody with existing config from having to provide this.

Reply:
All good - tested this out and was able to make it non-null with a default value to get the same behaviour. The config value can be omitted and will fallback to `true` (the way it works today).